### PR TITLE
fix lldb tests for Swift language changes

### PR DESCRIPTION
**Pull request by tfiala**
Fix up a couple LLDB swift-language tests that need adjustments based on Swift 3.0 preview-1 changes.

### CODE REVIEW

**Comment by tfiala** [comment]
@swift-ci Please test OS X platform

**Comment by tfiala** [comment]
Hmm, that build failure looks like an infrastructure issue.

Trying again.

@swift-ci Please test OS X platform

**Comment by tfiala** [comment]
@shahmishal, can you have a look?  The PR builder is failing with "Couldn't find cmark source directory.
":
```
Cloning into 'ninja'...
Cloning into 'swift'...
Switched to a new branch 'swift-3.0-preview-1-branch'
Branch swift-3.0-preview-1-branch set up to track remote branch swift-3.0-preview-1-branch from origin.
/Applications/Xcode-beta.app/Contents/Developer
Couldn't find cmark source directory.
/Users/buildnode/jenkins/workspace/swift-lldb-PR-osx/swift/utils/build-script: command terminated with a non-zero exit status 1, aborting
```

**Comment by shahmishal** [comment]
@swift-ci Please test OS X platform

**Comment by tfiala** [comment]
@shahmishal, there's a new unrelated failure:
```
--- Updating '/Users/buildnode/jenkins/workspace/swift-lldb-PR-osx/swift-corelibs-libdispatch' ---
Please, commit your changes.
 M libpwq

Build step 'Execute shell' marked build as failure
```

File: packages/Python/lldbsuite/test/lang/swift/po/sys_types/main.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ func main() {
   var any: Any = 1234 //% self.expect("po nsobject", substrs = ['<NSObject: 0x']) # may change depending on OS/platform
   //% self.expect("script lldb.frame.FindVariable('nsobject').GetObjectDescription()", substrs = ['<NSObject: 0x']) # may change depending on OS/platform
   var anyobject: AnyObject = 1234 as NSNumber //% self.expect("po any", substrs = ['1234'])
-  var notification = NSNotification(name: "JustANotification", object: nil)
+  var notification = NSNotification(name: "JustANotification" as NSNotification.Name, object: nil)
```
Comment (jrose-apple) [inline packages/Python/lldbsuite/test/lang/swift/po/sys_types/main.swift]: IIRC this should be Notification and Notification.Name.

**Comment by tfiala** [comment]
@jrose-apple, thanks for the pointer.  Notification worked in place of NSNotification.

@shahmishal - I've since verified this all locally.  I'm going to merge since the PR checker bot seems unhappy.

**Pull request merged**